src/Movie/Movie.js

```diff
@@ -6,13 +6,13 @@ const Movie = (props) => {
     <section className="movie-page">
```

Comment:
If you destructured your `props.movieSelected`, you'd save yourselves some typing:

`const { title, poster_path, average_rating, overview } = props.movieSelected`